apps/super-app/src/routes/SupportDetailWip.tsx

```diff
@@ -17,10 +18,24 @@ import './SupportDetailWip.css';
 
 function Header(props: { title: string }) {
   const { title } = props;
+  const navigate = useNavigate();
   return (
     <nav className="support-header">
-      <ArrowBackIosNewIcon />
-      <Typography variant="h4">{title}</Typography>
+      <IconButton
+        aria-label="back"
+        sx={{ color: 'text.primary' }}
+        onClick={() => navigate(-1)}
```

Comment:
As mentioned in the weekly meeting, I think we can merge this PR with `navigate(-1)` if we leave a TODO to check the edge cases with UXers.

Reply:
Left a TODO as discussed.